RELEASE_NOTES.md

```diff
@@ -7,6 +7,8 @@ This page describes the noteworthy improvements provided by each release of Ecli
 #### Editing improvements
   * XML auto closing tag
   * HTML auto closing tag and auto creation of quotes for HTML attribute assignment
+  * Syntax coloration / language configuration for RelaxNG compact syntax (.rng)
+  * Defines content type for .rnc, .rng RelaxNG files
```

Comment:
@mickaelistria lemminx is now released so if you embed it in wwd you should have a relaxng support like xsd and DTD inside xml. If you are OK I can change this text with relaxng support